### PR TITLE
finish use case 6

### LINKED ISSUE #9: MusicBox with non-standard loss rates file (Use Case 6)
### Use case 6: Simple box model with non-standard loss rates

This scenario is the same as Use Case 5, except that the user has been provided a list of wall-loss rates for the chamber from a colleague. The loss rates are in a non-standard text file named `wall_loss_rates_011519.txt` that is copied to the `configuration/` folder and contains:
```
simtime; 0.0; 0.3, 0.6, 0.9, 1.2
ISOP; 0.010, 0.012, 0.013, 0.014, 0.015
NO; 0.015, 0.016, 0.017, 0.018, 0.019
NO2; 0.015, 0.016, 0.017, 0.018, 0.019
MONO; 0.010, 0.012, 0.013, 0.014, 0.015
SESQ; 0.010, 0.012, 0.013, 0.014, 0.015
```

The times are in hours and the loss rates are in min-1. The `my_config.json` is updated to:
```
{
  “box model options” : {
    “grid”                      : “box”,
    “chemistry time step [min]” : 5.0,
    “output time step [hr]”     : 1.0,
    “simulation length [hr]”    : 2.5
  },
  “initial conditions” : {
    “init_cond.csv” : { 
      “delimiter” : “&”
    }
  },
  “photolysis” : {
    “O3 O1D” : { “initial value [s-1]” : 0.0032 },
    “O3 O3P” : { “initial value [s-1]” : 0.0012 }
  },
  “evolving conditions” : {
    “emissions.csv” : { },
    “wall_loss_rates_011519.txt” : {
      “delimiter” : “;”,
      “time axis” : “columns”,
      “properties” : {
        “simtime” : {
          “MusicBox name” : “time”,
          “units” : “hr”
        },
        “*” : {
          “MusicBox name” : “LOSS.*”,
          “units” : “min-1”
        }
      }
    }
  }
}
```

NOTES:
- The non-standard delimiter and time axis are specified in the configuration file.
- The simtime column is mapped to the MusicBox time and flagged as being in hours.
- The mapping of every property in the wall loss rates file is first attempted with the prefix `LOSS.`, which MusicBox identifies as a loss rate. If the mapping is successful, the units are specified as being in min-1. If the mapping is unsuccessful (e.g., if pressure or temperature had been specified in this file), then the original name (without the prefix) is used to do the mapping, with standard units.

### PR DESCRIPTION
**Pull request by mattldawson**
close #9

### CODE REVIEW

File: test/integration/input_use_cases/6/run.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# exit on error
+set -e
+# turn on command echoing
+set -v
+# make sure that the current directory is the once where this script is
+cd ${0%/*}
+
+exec_str="../../../../music_box config.json"
```
Comment (AndrewJConley) [inline test/integration/input_use_cases/6/run.sh:10]: Should paths be hard-coded or should they be abstracted: exec_str = $exectuble + "config.json"?

File: apps/music_box_core.F90 [hunk 2]
```diff
@@ -95,6 +98,7 @@ module music_box_core
   !! Loads input data and initializes model components.
   function constructor( config_file_path ) result( new_obj )
```
Comment (AndrewJConley) [inline apps/music_box_core.F90:99]: I wonder if there is a better name than new_obj.

File: test/integration/input_use_cases/6/run.sh (hunk 1 above)
Comment (mattldawson) [reply test/integration/input_use_cases/6/run.sh:10]: I'm not sure I follow - what would set `executable`?

File: apps/music_box_core.F90 (hunk 2 above)
Comment (mattldawson) [reply apps/music_box_core.F90:99]: probably - any suggestions?

**Pull request merged**